examples/karaf-dump-example/README.md

```diff
@@ -0,0 +1,49 @@
+# Apache Karaf Diagnostic Dump example
+
+## Abstract
+
+This example shows how to create a new dump provider service. This provider is automatically included in the dump created
+by the `dev:dump-create` command.
+
+The dump provider example actually takes a screenshot of client screen.
+
+## Build
+
+The build uses Apache Maven. Simply use:
+
+```
+mvn clean install
+```
+
+## Deployment
+
+On a running Karaf instance, install the dump bundle:
+
+```
```

Comment:
We have to install scr feature before install the bundle, we should add this in the documentation
```
feature:install scr
````